aroquev/src/main/webapp/script.js

```diff
@@ -346,3 +346,86 @@ function deleteAllComments() {
     getServletComments();
   });
 }
+
+function onDOMLoad() {
+  buildWriteCommentsBox();
+  getServletComments();
+}
+
+async function checkLoggedIn() {
+  const response = await fetch('/login-status');
+  const loginInfo = await response.json();
+  return loginInfo;
+}
+
+async function buildWriteCommentsBox() {
```

Comment:
This function is getting very long. Consider breaking it into smaller functions with distinct responsibilities.

Reply:
I tried to break it apart into two other functions that build the comment box depending on if the user is logged in or not.